Automattic/wp-calypso
# Site Settings: Get rid of component directories where obsolete

While developing Jetpack Settings in Calypso we inserted many of the components in their own folder, because they had own styles at some point. This is no longer true for several of these components, so I felt it's better if we leave them in the main `site-settings` directory. So this PR moves them to the root directory.

To test:
* Checkout this branch
* Verify the following settings cards are loaded properly in Calypso:
 * Writing > Media Settings
 * Writing > Theme Enhancements
 * Discussion > Subscriptions
 * General > Start Over
* Test a module toggle and verify it works like before.
